src/app/drive/services/file.service/uploadFile.ts

```diff
@@ -71,6 +71,7 @@ export async function uploadFile(
       bucket: bucketId,
       folder_id: file.parentFolderId,
       encrypt_version: StorageTypes.EncryptionVersion.Aes03,
+      plain_name: null
```

Comment:
Be careful with this, maybe @CandelR or @fabioespinosa could say something about